task/04-arrays-tasks.js

```diff
@@ -304,15 +318,15 @@ function get3TopItems(arr) {
  *   [ 1, '2' ] => 1
  */
 function getPositivesCount(arr) {
-   throw new Error('Not implemented');
+  return arr.filter(item => typeof item !== "string" && item > 0).length;
```

Comment:
JFYI: use braces in each `for`/`while`/`do ... while`/`if` operator or `callback`. You will make less mistakes.